recipes/mlt/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "mlt" %}
+{% set version = "6.18.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mltframework/mlt/releases/download/v{{ version }}/mlt-{{ version }}.tar.gz
+  sha256: fd2a346e0cc093e5d813e04cd126c077a2d224ae4d17b5226ff7478feaa445ea
+  patches:
+    # fix for GCC detection
+    - configure.patch
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - pkg-config
+  host:
+    - libxml2
+    - libvorbis
+    - ffmpeg
+    - lame
+    - alsa-lib
+    - pulseaudio
+    - ladspa
+    - jack
+    - fftw
+  run:
+    - libxml2
+    - jack
+    - fftw
```

Comment:
fftw has a run export now so you can remove this.

Reply:
Ok